cluster.tf

```
@@ -99,7 +100,7 @@ resource "aws_security_group_rule" "cluster_egress_internet" {
   description       = "Allow cluster egress access to the Internet."
   protocol          = "-1"
   security_group_id = local.cluster_security_group_id
-  cidr_blocks       = ["0.0.0.0/0"]
+  cidr_blocks       = ["0.0.0.0/0"] #tfsec:ignore:AWS007
```

this can also be written like:
```suggestion
  #tfsec:ignore:AWS007
  cidr_blocks       = ["0.0.0.0/0"]
```